### PR TITLE
Add planning_pipeline_id setting to Python MGI

### PR DESCRIPTION
**Pull request by felixvd**
This does what the title says.

### CODE REVIEW

**Comment by v4hn** [comment]
Guess it's time for you to setup pre-commit on your system? ;)

File: moveit_commander/src/moveit_commander/move_group.py [hunk 1]
```diff
@@ -532,12 +532,20 @@ def get_planning_time(self):
         """ Specify the amount of time to be used for motion planning. """
         return self._g.get_planning_time()
 
+    def set_planning_pipeline_id(self, planning_pipeline):
+        """ Specify which planning pipeline to use when motion planning (e.g. OMPL, pilz_industrial_motion_planner) """
+        self._g.set_planning_pipeline_id(planning_pipeline)
```
Comment (v4hn) [inline moveit_commander/src/moveit_commander/move_group.py:537]: I guess `ompl` should not be in capitals here?

File: moveit_commander/src/moveit_commander/move_group.py [hunk 2]
```diff
@@ -532,12 +532,20 @@ def get_planning_time(self):
...
+
+    def get_planning_pipeline_id(self, planning_pipeline):
+        """ Get the current planning_pipeline_id (e.g. OMPL, pilz_industrial_motion_planner) """
```
Comment (v4hn) [inline moveit_commander/src/moveit_commander/move_group.py:540]: same as above.

File: moveit_commander/src/moveit_commander/move_group.py [hunk 3]
```diff
@@ -532,12 +532,20 @@ def get_planning_time(self):
...
+        self._g.get_planning_pipeline_id(planning_pipeline)
+
     def set_planner_id(self, planner_id):
-        """ Specify which planner to use when motion planning """
+        """ Specify which planner to use when motion planning (e.g. RRTConnect, LIN) """
```
Comment (v4hn) [inline moveit_commander/src/moveit_commander/move_group.py:544]:
```suggestion
        """ Specify which planner of the selected pipeline to use when motion planning (e.g. RRTConnect, LIN) """
```

File: moveit_commander/src/moveit_commander/move_group.py [hunk 4]
```diff
@@ -532,12 +532,20 @@ def get_planning_time(self):
...
         self._g.set_planner_id(planner_id)
 
     def get_planner_id(self):
-        """ Get the current planner_id """
+        """ Get the current planner_id (e.g. RRTConnect, LIN) """
```
Comment (v4hn) [inline moveit_commander/src/moveit_commander/move_group.py:548]:
```suggestion
        """ Get the current planner_id of the current pipeline (e.g. RRTConnect, LIN) """
```

File: moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp [hunk 5]
```diff
@@ -417,6 +417,12 @@ class MoveGroupInterfaceWrapper : protected py_bindings_tools::ROScppInitializer
     return getPlannerId().c_str();
   }
 
+  const char* getPlanningPipelineIdCStr() const
+  {
+    return getPlanningPipelineId().c_str();
+  }
+  
+
```
Comment (v4hn) [inline moveit_ros/planning_interface/move_group_interface/src/wrap_python_move_group.cpp:425]:
```suggestion

```

**Review by v4hn — APPROVED** [review]
looks good.

Please review my inline comments

**Comment by felixvd** [comment]
Good catches. Yes, I didn't have pre-commit on the machine where I originally submitted this from. Should be fine now.

**Comment by felixvd** [comment]
Is this a flaky test?

**Comment by v4hn** [comment]
Seems like it. If we see it again, someone should probably investigate.

**Pull request merged**